src/routes/error-router.ts

```diff
@@ -63,13 +69,27 @@ ErrorRouter.use((err: Error, req: Request, res: Response, next: NextFunction) =>
 		"Not Found": 404
 	};
 
-	const messages = {
-		[Errors.MISSING_JIRA_HOST]: "Session information missing - please enable all cookies in your browser settings.",
-		[Errors.IP_ALLOWLIST_MISCONFIGURED]: `The GitHub org you are trying to connect is currently blocking our requests. To configure the GitHub IP Allow List correctly, <a href="${envVars.GITHUB_REPO_URL}/blob/main/docs/ip-allowlist.md">please follow these instructions</a>.`
-	};
+	let message: string | null = null;
+	let ctaUrl: Cta | null = null;
```

Comment:
Couple of things here:
- The text also shouldn't be defined on the backend as the backend should have no knowledge of what is rendered in the UI. Create the url and then set the text in the UI (there's no need for it to be passed from the backend).
- I think ctaUrl should be defined directly set it in the render function: `ctaUrl: createUrlWithQueryString(req, "/session")`.

Reply:
The call to action URL is only for the _MISSING_GITHUB_APP_NAME_ error. So, if we go by this approach, we'll have to add another condition just to add `ctaUrl`. I wanted to avoid having to do this separate condition.